Composer/packages/client/setupTests.ts

```diff
@@ -23,3 +23,6 @@ expect.extend({
     };
   },
 });
+
+// for tests using Electron IPC to talk to main process
+(window as any).ipcRenderer = { on: jest.fn() };
```

Comment:
is there a jest env setting that would add this?

Reply:
unfortunately not =/